recipes/spyder-vim/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "spyder-vim" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spyder-vim-{{ version }}.tar.gz
```

```suggestion
  url: https://pypi.io/packages/source/{{ name[0]|lower }}/{{ name|lower }}/{{ name|lower }}-{{ version }}.tar.gz
```

Missed a spot. Also, I don't think this is case sensitive, but just in case...